tests/data/data_structure_definition/subannual/subannual/months.yaml

```diff
@@ -0,0 +1,24 @@
+- January:
```

Comment:
I think that this test doesn't have any effect (yet), so I suggest to remove and only add when categorical time-domain validation is added explicitly.